mscore/prefsdialog.ui

```diff
@@ -4284,6 +4320,7 @@ Adjusting latency can help synchronize your MIDI hardware with MuseScore's inter
    <class>Ms::RadioButtonGroupBox</class>
    <extends>QGroupBox</extends>
    <header>radiobuttongroupbox.h</header>
+   <container>1</container>
```

Comment:
What is this change for?

Reply:
As it happens, I created this `Ms::RadioButtonGroupBox` control less than two weeks ago (see #5992 if you're interested), and I hand-edited this block of code to add it here, so I was surprised to see you making an unexplained change to it in an unrelated PR.

Not sure why you suggested that I read the Qt documentation instead of answering my question.

Reply:
Nice observation

Reply:
@Spire42 P.S., Check out PR #6506, as it looks looks like that line was added there as well with prefsdialog.ui. Again, maybe some Qt documentation might be able to answer your question, as you give the impression of wanting to understand why the <container>1</container> line is being added